homeassistant/components/homekit/__init__.py

```diff
@@ -99,6 +99,8 @@ def get_accessory(hass, state, aid, config):
         if device_class == 'garage' and \
                 features & (SUPPORT_OPEN | SUPPORT_CLOSE):
             a_type = 'GarageDoorOpener'
+        elif features & (SUPPORT_OPEN | SUPPORT_CLOSE):
+            a_type = 'WindowCoveringBasic'
```

Comment:
Please put this case below the `support_set_position` check. Otherwise a cover that allows for open, close and position would be limited to the basic case.